README.md

```diff
@@ -76,59 +48,33 @@ const searchResult = await shopperSearch.productSearch({
 });
 ```
 
-#### CORS
-
-The Salesforce Commerce API (SCAPI) does not support CORS, so a proxy must be used to be able to use the SDK.
+#### Fetch Options
 
-### Advanced Options
-
-Commerce SDK Isomorphic supports Fetch API options for [node-fetch](https://github.com/node-fetch/node-fetch/1#api) on server and [whatwg-fetch](https://github.github.io/fetch/) on browser with a simple configuration.
-This sample code shows how to configure HTTP timeout and agent options.
+You can configure how the SDK makes requests using the `fetchOptions` parameter. It is passed to [node-fetch](https://github.com/node-fetch/node-fetch/1#api) on the server and [whatwg-fetch](https://github.github.io/fetch/) on browser.
 
 ```javascript
-/**
- * Configure advanced timeout and agent parameters
- *
- * To learn more about the parameters please refer to the [Salesforce Developer Center](https://developer.salesforce.com/docs/commerce/commerce-api).
- */
-// Create a configuration to use when creating API clients
-const https = require('https');
+const https = require("https");
 
 const config = {
-  proxy: 'https://localhost:3000',
-  headers: {},
-  parameters: {
-    clientId: '<your-client-id>',
-    organizationId: '<your-org-id>',
-    shortCode: '<your-short-code>',
-    siteId: '<your-site-id>',
-  },
   fetchOptions: {
-    timeout: 2000, //request times out after 2 seconds
-    agent: new https.agent({
-      // a custom http agent
-      keepAlive: true,
-    }),
+    // By default, requests made using the SDK do not include cookies.
+    credentials: "include",
+    timeout: 2000,
+    agent: new https.agent({ keepAlive: true }),
   },
 };
 ```
 
-### Additional Config Settings
-
-_headers:_ A collection of key/value string pairs representing additional headers to include with API requests.
+For more info, refer to the [documentation site](https://salesforcecommercecloud.github.io/commerce-sdk-isomorphic/).
 
-_throwOnBadResponse:_ Default value is false. When set to true, the SDK throws an Error on responses with statuses that are not 2xx or 304.
+#### Additional Config Settings
 
-### Public/Private Client Shopper Login (SLAS) helpers
-
-A collection of helper functions are available in this SDK to simplify [Public/Private Client Shopper Login OAuth flows](https://developer.salesforce.com/docs/commerce/commerce-api/references?meta=shopper-login:Summary). See sample code above for guest login.
-
-**⚠️ WARNING ⚠️**
-Users should be extremely cautious about using the SLAS private client helper functions in the browser as it can expose your client secret. Ensure that your client secret is secured before running the function client side.
```

Comment:
I think we should consider leaving in this warning somewhere about SLAS private client and potentially exposing it on the client side

Reply:
@joeluong-sfcc – I think using a Client Secret represents a very advanced use case that I wouldn't document in the README.

The function itself has this warning:

https://salesforcecommercecloud.github.io/commerce-sdk-isomorphic/modules/helpers.html#loginguestuserprivate-1

I don't feel strongly.